### ISSUE TITLE
Replace absolute paths with relative paths for outside continuous integration

### ISSUE THREAD

**Issue by russelljjarvis** [russelljjarvis/teili#2]
Hi @pabloabur,

I was wondering if you could replace absolute paths with relative paths for outside continuous integration

Here is a list of absolute paths. I am not sure if the files at these paths are staged? Is this a reading file or a writing file? If it is only for writing perhaps it doesn't matter. It does look like these models are for reading though.

tutorials/SLIF_rotating_bar.py

77 | Net = TeiliNetwork()
-- | --
```
78 | model_path = '/Users/Pablo/git/teili/'
```
79 | layer='L4'
80 | conn_desc = ConnectionDescriptor(layer, model_path)

</div></div></div>[tutorials/SLIF_rotating_bar.py](https://github.com/russelljjarvis/teili/blob/5dc1f88f8edb19406ae107a06935283022425435/tutorials/SLIF_rotating_bar.py)
Net = TeiliNetwork()
```
model_path = '/Users/Pablo/git/teili/'
```
layer='L4'
conn_desc = ConnectionDescriptor(layer, model_path)
 Python
Showing the top match
Last indexed on 6 Apr
[tutorials/SLIF_sequence_learning.py](https://github.com/russelljjarvis/teili/blob/5dc1f88f8edb19406ae107a06935283022425435/tutorials/SLIF_sequence_learning.py)
                                       spike_indices=input_indices,
                                       spike_times=input_times)

Net = TeiliNetwork()
```
model_path = '/Users/Pablo/git/teili/'
```
 Python
Showing the top match
Last indexed on 6 Apr
tutorials/SLIF_balanced_tutorial.py
poisson_spikes = PoissonGroup(2[85](https://github.com/russelljjarvis/teili/blob/5dc1f88f8edb19406ae107a06935283022425435/tutorials/SLIF_sequence_learning.py#L85), rates=6*Hz)

layer = 'L4'
```
path = '/Users/Pablo/git/teili/'
```
conn_desc = ConnectionDescriptor(layer, path)
pop_desc = PopulationDescriptor(layer, path)
 Python
Showing the top match
Last indexed on 6 Apr
tutorials/SLIF_istdp_tutorial.py
```
path = os.path.expanduser('/Users/Pablo/git/teili/')
```
model_path = os.path.join(path, "teili", "models", "equations", "")
 Python

**Comment by pabloabur** [comment]
Thanks! That makes perfect sense. When teili is being installed, a folder ~/teili_apps is created, so I'd rather put all files supposed to be read in there, right?

**Comment by russelljjarvis** [comment]
Yes, that would be awesome. the ~/teili_apps folder would be a good relative path.

Oh hang on: what about two options:

* if git/teili exists because package was `git cloned` that should be first most natural choice.
* if package was pip installed use ~/teili_apps instead.

But really any sensible relative path will help me reproduce your work.

**Comment by pabloabur** [comment]
I pushed some changes with no such relative paths. They still exist, but I just want them there for now because I need to re-write (I know, it's terrible). Also, some tutorials still need to be updated, so if they use relative path, they're not ready yet
